spec/data-model/README.md

```diff
@@ -103,23 +106,25 @@ interface Variant {
 
 interface CatchallKey {
   type: "*";
-  value?: string;
+  value: Pattern;
 }
 ```
 
 ## Patterns
 
-Each `Pattern` contains a linear sequence of text and placeholders corresponding to potential output of a message.
+Each `Pattern` contains a linear sequence of text and placeholders corresponding
+to potential output of a message.
 
-Each element of the `Pattern` MUST either be a non-empty string, an `Expression`, or a `Markup` object.
-String values represent literal _text_.
+Each element of the `Pattern` MUST either be a non-empty string, an `Expression`,
+or a `Markup` object. \
```

Comment:
```suggestion
or a `Markup`.
```
Delete "object" for consistency with the rest of the sentence (it doesn't say "an `Expression` object").

And, I don't think the backslashes are necessary? (Here and elsewhere.)